Makefile

```diff
@@ -803,6 +850,15 @@ $(CONVERSION_GEN): # Build conversion-gen from tools folder.
 $(CONVERSION_VERIFIER): $(TOOLS_DIR)/go.mod # Build conversion-verifier from tools folder.
 	cd $(TOOLS_DIR); go build -tags=tools -o $(BIN_DIR)/$(CONVERSION_VERIFIER_BIN) sigs.k8s.io/cluster-api/hack/tools/conversion-verifier
 
+.PHONY: $(OPENAPI_GEN)
```

Comment:
Please add the same comment here as in l.844-845

Reply:
nope, won't do after changing to versioned binary